backend/clubs/serializers.py

```diff
@@ -1736,7 +1736,10 @@ def get_clubs(self, obj):
         # hide non public memberships if not superuser
```

Comment:
@cphalen I would also hide this from superusers and only allow it on the Django admin console, to prevent confusing OSA.

Reply:
That's an excellent point thanks